### PR TITLE
[Task]: Remove extracted Web2Print bundle from Core repository

### LINKED ISSUE #12525: [Task] Move Web-to-print to repository pimcore/web-to-print-bundle
### Improvement description

Move `Web-to-print` to repository `pimcore/web-to-print-bundle`

Related and depends on https://github.com/pimcore/pimcore/pull/14047

### PR DESCRIPTION
**Pull request by kingjia90**
## Changes in this pull request  
Resolves https://github.com/pimcore/pimcore/issues/12525

See https://github.com/pimcore/web-to-print-bundle

### CODE REVIEW

File: doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md [hunk 1]
```diff
@@ -150,7 +150,7 @@
     - [UUID] has been moved into PimcoreUuidBundle
       - Config `pimcore:general:instance_identifier` has been removed, please use `pimcore_uuid:instance_identifier` in the PimcoreUuidBundle instead. Please run `bin/console config:dump pimcore_uuid` to see more about the instance identifier config after installing the bundle.
     - [Application Logger] Application logger has been moved into `PimcoreApplicationLoggerBundle`. Please pay attention to the new namespaces for the classes from this bundle.
-    - [Web2Print] has been moved into PimcoreWebToPrintBundle
+    - [Web2Print] has been moved to PimcoreWebToPrintBundle in an own [repository](https://github.com/pimcore/web-to-print-bundle).
```
Comment (mattamon) [inline doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]:
```suggestion
    - [Web2Print] has been moved to `pimcore/web-to-print-bundle` package.
```

Stay consistent with the wording for other repos extractions

**Review by mattamon — CHANGES_REQUESTED** [review]
Rest looks fine to me :)

**Review by mattamon — APPROVED** [review]

**Comment by mattamon** [comment]
Now we should resolve the conflicts first :)

**Comment by kingjia90** [comment]
@brusch there were some further changes after the extraction, i've "collected" them in https://github.com/pimcore/web-to-print-bundle/pull/6 (when resolving conflicts). 

~Before merging the removal, shall we re-sync again or just merge that PR?~ we just merged the PR in the repo

**Pull request merged**